sbomtool/cmd/sbomtool/main.go

```diff
@@ -5,22 +5,89 @@ import (
 	"fmt"
 	"log/slog"
 	"os"
+	"path/filepath"
 
-	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/pkg/generate"
-	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/pkg/logging"
-	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/pkg/merge"
-	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/pkg/validate"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/sbom"
 	"github.com/spf13/cobra"
+
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/commands/filter"
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/commands/generate"
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/commands/merge"
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/deduplication"
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/logging"
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/processor"
+	"github.com/bottlerocket-os/bottlerocket-sdk/sbomtool/go/internal/validate"
+)
+
+// Exit codes
+const (
+	ExitSuccess = 0
+	ExitError   = 1
+)
+
+// Version information
+const (
+	Version   = "1.0.0"
+	BuildDate = "2025-08-04"
+	GitCommit = "dev"
 )
 
+// validationError represents an input validation failure with contextual guidance.
+type validationError struct {
+	Field      string
+	Value      string
+	Message    string
+	Suggestion string
+}
+
+func (e *validationError) Error() string {
```

Comment:
just a nit but it looks like the `Error()` impl for validationError and filesystemError has the same logic regarding the `suggestion` but slightly different implementations